snapshot-replicator/main.tf

```diff
@@ -62,6 +62,13 @@ resource "aws_iam_role_policy_attachment" "attach_lambda_copy_policy_to_role" {
   policy_arn = aws_iam_policy.rds_snapshot_copy[0].arn
 }
 
+resource "aws_iam_user_policy_attachment" "lambda_exec_role" {
+  count      = var.enable ? 1 : 0
+  provider   = aws.replica
```

Comment:
I think you should use the normal provider here

Reply:
ah indeed, copy/paste mistake! thanks for spotting that